spec/models/day_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Day, type: :model do
+  describe 'validations' do
+    it { should validate_presence_of(:week) }
+    it { should validate_presence_of(:day_of_week) }
```

Comment:
You can add tests for enums with `should define_enum_for`, that would confirm the enums are behaving as expected

Reply:
i like it